bundles/admin/admin-layereditor/components/Button.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import { Button as AntButton } from 'antd';
 import 'antd/es/button/style/index.js';
 
-export const Button = ({children, ...other}) => (
-    <AntButton {...other}>{children}</AntButton>
+const Styled = styled(AntButton)`
+    > .anticon {
```

Comment:
This is because we configured the ant-styles out of the global scope so we need to do some extra tweaking to make the icon align perfectly as with the global styles. Add something along these lines as a comment as the reasoning for this.

Reply:
Or should we gather some readme.md on the components folder to explain this and have some info about the styling in general as well (some mention about webpack config for these)?

Reply:
Fixed ant style handling by modifyin ant-globals.less. Also relocated ant-globals.less and ant-theme.less under oskari-ui.
Added README.md under oskari-ui.